chapters/en/slides/module7/module7_03.md

```diff
@@ -0,0 +1,389 @@
+in the options array show up in the widget, but it is not used for
+selection until we interact with the dropdown.
+
+If we wanted to set a specific default value and immediately use it to
+highlight the data, we could specify which column and value to use via
+the `init` parameter as we have done in this slide.
+
+---
+
+## Adding multiple widgets to the same charts
+
+``` python
+mpaa_rating = sorted(movies['MPAA_Rating'].unique())
+dropdown_mpaa = alt.binding_radio(name='MPAA Rating ', options=mpaa_rating)
+dropdown_genre = alt.binding_select(name='Genre ', options=genres)
+select_genre_and_mpaa = alt.selection_single(
+    fields=['Major_Genre', 'MPAA_Rating'],
+    bind={'Major_Genre': dropdown_genre, 'MPAA_Rating': dropdown_mpaa})
+
+points.add_selection(select_genre_and_mpaa).encode(
+    opacity=alt.condition(select_genre_and_mpaa, alt.value(0.7), alt.value(0.1)))
+```
+
+<iframe src="/module7/charts/03/unnamed-chunk-5.html" width="100%" height="420px" style="border:none;">
+</iframe>
+
+Notes: Sometimes we want to highlight our data based on more than a
+single widget. For example, we might want to show only comedies that are
+rated as suitable for children to watch.
+
+We have already seen how to perform such breakdowns via faceting, but we
+could also do it by combining multiple interactive widgets.
+
+Here we’re adding a set of radio buttons for the maturity rating of the
+movies and a dropdown button for the genre selection.
+
+We create the radio buttons and dropdown separately, and then add then
+bind them both to the `selection_single` object to their respective
+fields.
+
+To trigger a selection, we now need to both click a radio button **and**
+make a selection in the dropdown.
+
+---
+
+## Adding sliders allows us to highlight points based on quantitative value comparisons
+
+``` python
+slider = alt.binding_range(name='Tomatometer ')
+select_rating = alt.selection_single(
+    fields=['Rotten_Tomatoes_Rating'],
+    bind=slider)
+
+points.encode(
+    opacity=alt.condition(select_rating, alt.value(0.7), alt.value(0.1))
+).add_selection(select_rating)
+```
+
+<iframe src="/module7/charts/03/unnamed-chunk-6.html" width="100%" height="420px" style="border:none;">
+</iframe>
+
+Notes: So far we have seen how to select categorical values from
+dropdown menus and radio buttons. To instead highlight based on
+quantitative values, we can use a slider.
+
+Adding a slider is similar to adding a dropdown or set of radio buttons,
+and we do it via `binding_range()`. Here we link the slider to the same
+column as on the x-axis, so that you can see how it works.
+
+If you drag the slider around, you can see that the default behavior is
+to only highlight points that are of the exact values as the slider.
+
+This is useful for selection widgets like the dropdown, but for the
+slider we want to make comparisons such as bigger and smaller than.
+
+Let’s see how to do that in the next slide.
```

Comment:
```suggestion
Notes: So far we have seen how to select categorical values from
dropdown menus and radio buttons. To instead highlight based on
quantitative values, we can use a slider.

Adding a slider is similar to adding a dropdown or set of radio buttons,
and we do it via `binding_range()`. Here we link the slider to the same
column that is mapped to the x-axis, so that you can see how it works.

If you drag the slider around, you can see that the default behaviour is
to only highlight points that share the x-column values that are of the 
exact values as the slider - this is because they are both mapped to the 
`Rotten_Tomatoes_Rating` column.

Now that we understand how we can create a slider, let's next see how to  
use it in a slightly more realistic setting, such as highlighting values above,
or below a threshold.
```